src/pages/MsigUI.vue

```diff
@@ -161,28 +147,30 @@ export default defineComponent({
 
 			try {
```

Comment:
This file is named `MsigUI.vue` -- not sure if this is the right convention. @Megha-Dev-19 could you please check?

Reply:
Ideally we should avoid putting multiple PR's in the pipeline because of this. But my original question was is this casing/naming convention correct?

Reply:
okay will add a task for it